src/fiskaltrust.Launcher/Extensions/LoggerConfigurationExtensions.cs

```diff
@@ -19,16 +19,20 @@ public static LoggerConfiguration AddLoggingConfiguration(this LoggerConfigurati
                 suffix = "";
             }
 
-            return loggerConfiguration.MinimumLevel.Is(Serilog.Extensions.Logging.LevelConvert.ToSerilogLevel(launcherConfiguration.LogLevel))
-            .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}{EnrichedPackage}{EnrichedId}] {Message:lj}{NewLine}{Exception}")
+            return loggerConfiguration.MinimumLevel.Is(Serilog.Extensions.Logging.LevelConvert.ToSerilogLevel(launcherConfiguration.LogLevel!.Value))
+            .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}{EnrichedPackage}{EnrichedId}{EnrichedContext}] {Message:lj}{NewLine}{Exception}")
             .MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
             .MinimumLevel.Override("System", LogEventLevel.Warning)
             .MinimumLevel.Override("Grpc", LogEventLevel.Warning)
             .MinimumLevel.Override("ProtoBuf", LogEventLevel.Warning)
             .WriteTo.Logger(configuration =>
                 configuration
                     .Filter.ByExcluding(Matching.WithProperty("EnrichedId"))
-                    .WriteTo.File(Path.Join(launcherConfiguration.LogFolder, $"log{suffix}-.txt"), rollingInterval: RollingInterval.Day, shared: true));
+                    .WriteTo.File(
```

Comment:
Should we maybe add a `retainedFileCountLimit` of e.g. 60 days?

Reply:
the default `retainedFileCountLimit ` is 31 files and `retainedFileTimeLimit` is none 🤷